File: Decorator/C#/CoffeeShopNoDecorator/HouseBlendWithMilk.cs
namespace CoffeeShopNoDecorator
{
    public class HouseBlendWithMilk : Beverage
    {
        public HouseBlendWithMilk()
        {
            Description = "House blend with milk";
        }

        public override decimal GetPrice()
        {
            return 1.3m + 0.2m;
        }
    }
}

File: Decorator/Java/CoffeeShopNoDecorator/src/CoffeeShopNoDecorator/HouseBlendWithMilk.java
package CoffeeShopNoDecorator;

public class HouseBlendWithMilk extends Beverage
{
    public HouseBlendWithMilk()
    {
        description = "House blend with milk";
    }

    @Override
    public double getPrice()
    {
        return 1.3d + 0.2d;
    }
}